src/js/modules/wallet/controllers/send-scan-qr/send-scan-qr.ctrl.js

```diff
@@ -50,8 +50,14 @@
                         throw new Error("Can't send to Bitcoin Cash address with BTC wallet");
                     } else if (elm.protocol === 'bitcoin:' || elm.protocol === 'bitcoincash:') {
                         $scope.clearRecipient();
-                        bitcoinLinkService.parse(result).then(function (sendParams) {
-                            $state.go('app.wallet.send', {sendInput: sendParams});
+                        bitcoinLinkService.parse(result).then(function (sendInput) {
+                            if(sendInput && (sendInput.network === 'bitcoin' || sendInput.network === 'bitcoincash')) {
+                                $state.go('app.wallet.send', {
+                                    sendInput: sendInput
+                                });
+                            } else {
+                                $state.go('app.wallet.summary');
```

Comment:
Shouldn't this go in the second closure, the one that handles the error?

https://github.com/blocktrail/blocktrail-wallet/blob/15e3595918b87811c78d39a5a167b6cd4898a44c/src/js/modules/core/services/bitcoin-link/bitcoin-link.service.js#L79 
also since this is here, we can remove this line